### PR TITLE
Unit 4: Standards and Guidelines -- Live Draft

### PR DESCRIPTION
**Pull request by daniel-montalvo**

### CODE REVIEW

**Comment by daniel-montalvo** [comment]
Work on unit 4 "Standards and Guidelines" during survey will be stored in this deployed branch.

File: 1-3.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+permalink: /curricula/1-3/
+ref: /curricula/1-3/
+lang: en
+github:
+  repository: w3c/wai-curricula
+  path: 1-3.md
+footer: >
+  <p>Work in Progress.</p>
+  <p><strong>Date:</strong> $date </p>
+  <p><strong>Editors:</strong>Daniel Montalvo and <a href="http://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Contributors: <a href="https://www.w3.org/WAI/EO/EOWG-members">EOWG Participants</a>. </p>
```
Comment (nitedog) [inline 1-3.md]: Please add spacing between "</strong>" and "Daniel"

File: 1-3.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+navigation:
+  previous: /curricula/1-2/
+  next: /curricula/1-4/
+---
+
+## Introduction
+{:.no-display}
+ 
+Courses based on this unit are expected to:
+* examine the rationale for organizations to address accessibility
```
Comment (nitedog) [inline 1-3.md]: Not sure "examine" makes sense for a course? Maybe "explore"?

File: 1-3.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+* present different testimonials from organizations that have addressed accessibility
+
+Note: Introduce different case studies for accessibility in organizations. Select them as a starting point for students to build their own business case arguments. Use the resource [The Business Case for Digital Accessibility](https://www.w3.org/wai/business-case/).
```
Comment (nitedog) [inline 1-3.md]: This statement seems very instructive without any indication of how to do that. Maybe add something "Research relevant case studies on accessibility in organizations that are relevant to your audience"? Seems like a tall order in any case (where to research and how?)

File: 1-3.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+## Learning Outcomes
+
+Students should be able to:
+* Identify accessibility as a driving force for innovation in products
```
Comment (nitedog) [inline 1-3.md]: Maybe changing "driving force" to just "driver"? I don't feel strongly.

File: 1-3.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+* Recognize accessibility as a discipline that promotes  inclusion
+* Explain how accessibility can extend market reach for organizations
+* Identify how to minimize legal risk for some organizations
+
+## Competencies
+
+The following is a list of skills required for this unit.
```
Comment (nitedog) [inline 1-3.md]: Maybe drop "The following is a list of" and just keep "Required skills:"?

File: 1-3.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+
+{% include excol.html type="all" %}
+
+{% include excol.html type="start" %}
+
+### Topic: Accessibility as a Drive for Innovation
+
+{% include excol.html type="middle" %}
+
+Explain how accessibility helps organizations remove architectural, digital, and social barriers. Use the resource [Business Case for Digital Accessibility (Drive Innovation)](https://www.w3.org/WAI/business-case/#drive-innovation).
```
Comment (nitedog) [inline 1-3.md]: Slight discomfort with "architectural" -- seems like scope creep and we should focus on digital accessibility here (comparisons and analogies are fine but here you as instructors to *explain* these aspects)

File: 1-3.md [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+{% endcomment %}
+
+#### Teaching Ideas
+
+* Discuss how accessibility solutions are also enjoyed by many people regardless of their disability. Emphasize how many solutions originally thought for accessibility have become mainstream, such as the type-writer, telephone, punch cards, text to speech, email, and voice controls.
+* Promote discussion among students about which aspects of accessibility benefit them as well as people with disabilities. For instance, ask them which products they can use without looking at them or without pressing any button.
+
+#### Homework Ideas
+
+* Ask students to research how many people with disabilities there are in their organization. Emphasize that statistics on this matter are difficult to find and interpret. The concept of disability, the way it is measured in different cultures, and the right for people not to disclose their disabilities may influence the results.
```
Comment (nitedog) [inline 1-3.md]: Hmmm - could be a privacy concern going around and asking people to disclose if they have a disability. Maybe "ask students to think of examples of people with disabilities they know of" or such?

File: 1-3.md [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+
+### Topic: Broader Benefits of Accessibility
+
+{% include excol.html type="middle" %}
+
+Introduce other benefits of accessibility, such as technical or financial. Use the resource [Developing an Accessibility Business Case for your Organization (Archived)](https://www.w3.org/WAI/business-case/archive/).
+
+#### Learning Outcomes
+
+* Explain how accessibility can enhance the brand perception and contribute to some other benefits related to CSR
```
Comment (nitedog) [inline 1-3.md]: What is CSR? ;-)

File: 1-3.md [hunk 9]
```diff
@@ -0,0 +1,164 @@
...
+* Identify some international, regional, and local accessibility standards and policies
+
+#### Teaching Ideas
+
+* Engage students in a conversation about how they could improve access to information for people with disabilities.
+* Ask students to identify which policies may apply to their organizations. Go through the resource [Web Accessibility Laws & Policies](https://www.w3.org/wai/policies/).
+
+#### Homework Ideas
+
+* Ask students to research on the  [Convention on the Rights of Persons with Disabilities (CRPD)](https://www.un.org/development/desa/disabilities/convention-on-the-rights-of-persons-with-disabilities.html). Encourage them to share with their colleagues the conception that it offers about people with disabilities. Concentrate on the right to access to information found in article 9.
```
Comment (nitedog) [inline 1-3.md]: Research what?
